apps/terraform/terraform.talon

```diff
@@ -0,0 +1,11 @@
+tag: terminal
+and tag: user.terraform
+-
+terraform: "terraform "
+
+terraform init: "terraform init\n"
+terraform init upgrade: "terraform init -upgrade\n"
+terraform plan: "terraform plan\n"
+terraform apply: "terraform apply "
+terraform destroy: "terraform destroy "
+terraform state move: "terraform state mv "
```

Comment:
@2d6 could you, please, also add `fmt` and `validate` commands as well?
Thanks in advance.